### PR TITLE
win_timezone: Add RETURN information

### LINKED ISSUE #1767: New Windows Modules Missing RETURN docs
##### Issue Type:

Documentation Report
##### Plugin Name:
- `win_regmerge`
- `win_timezone`
##### Ansible Version:

```
v2.1
```
##### Ansible Configuration:

N/A
##### Environment:

N/A
##### Summary:

cc @jhawkesworth @schwartzmx 

The 2 new merged modules are missing RETURN documentation which is required for all new modules.

This is an issue to track addition of this information.
##### Steps To Reproduce:

For bugs, please show exactly how to reproduce the problem. For new
features, show how the feature would be used.

```
(Paste example playbooks or commands here)
```

You can also paste gist.github.com links for larger files.
##### Expected Results:

N/A
##### Actual Results:

N/A

### PR DESCRIPTION
**Pull request by dagwieers**
##### SUMMARY

And cosmetic cleanup.

This fixes ansible/ansible-modules-extras#1767

##### ISSUE TYPE

 - Docs Pull Request

##### COMPONENT NAME

win_timezone

##### ANSIBLE VERSION

v2.4

### CODE REVIEW

**Comment by jborean93** [comment]
shipit

**Comment by pchauncey** [comment]
works for me, tested with Ansible 2.4.0.  shipit

**Comment by pchauncey** [comment]
shipit

**Review by nitzmahone — APPROVED** [review]

File: lib/ansible/modules/windows/win_timezone.ps1 [hunk 1]
```diff
@@ -19,16 +19,15 @@
...
-    changed = $false
-    win_timezone = @{}
-}
-
 $params = Parse-Args $args -supports_check_mode $true
 $check_mode = Get-AnsibleParam -obj $params -name "_ansible_check_mode" -type "bool" -default $false
 
 $timezone = Get-AnsibleParam -obj $params -name "timezone" -failifempty $true -resultobj $result
 
+$result = @{
```
Comment (nitzmahone) [inline lib/ansible/modules/windows/win_timezone.ps1]: bang- this is getting used in the line above

**Review by nitzmahone — CHANGES_REQUESTED** [review]

File: lib/ansible/modules/windows/win_timezone.ps1 (hunk 1 above)
Comment (dagwieers) [reply lib/ansible/modules/windows/win_timezone.ps1]: Why oh why. I never really understood the purpose of the -resultobj option in Get-AnsibleParam. And especially in this case there's no purpose.

**Review by nitzmahone — APPROVED** [review]

**Pull request merged**